recipe/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Watermelon Margarita</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <div class="top"> 
+    <h1> Watermelon Margarita </h1>
+    <p> This watermelon margarita recipe is the best drink you'll make all summer! With no added sweetener and just 5
+      ingredients, it's healthy and fun. </p>
+  </div>
+
+  <div class="left">
+    <div>
+      <img src="https://cdn.loveandlemons.com/wp-content/uploads/2018/08/watermelon-margarita-recipe.jpg"
+        alt="Crystal glass containing red alcoholic drink" height="600px">
+      <a href="https://www.loveandlemons.com/watermelon-margarita/"> Visit Here for Original Image </a>
+    </div>
+  </div>
+  <div>
+    <h2> Ingredients </h2>
+
+    <ul>
+      <li> ½ cup tequila </li>
+      <li> 1 small jalapeño, thinly sliced (optional) </li>
+      <li> 1/4 cup lime juice, plus 4 sliced lime rounds for garnish</li>
+      <li> Handfuls of ice </li>
+      <li> Sea salt for glass rims, optional </li>
+    </ul>
+  </div>
+  </div>
+
+  <div class="right">
+    <div>
+      <h2> Instructions </h2>
+
+      <ol>
+        <li> Combine the tequila and half of the jalapeño slices, if using. Set aside for 15 to 30 minutes to infuse
+          (the
+          longer you infuse, the spicier it will be). If you're not using the jalapeño, skip this step and proceed
+          with the
+          recipe.
+        </li>
+        <li> In a blender, combine the watermelon with the lime juice and a handful of ice, and blend until smooth.
+        </li>
+        <li> Salt the rim of 4 glasses, if desired, and fill each with ice. Pour the watermelon lime juice into the
+          glasses
+          and stir 2 tablespoons of tequila into each glass. Garnish with the lime rounds.</li>
+      </ol>
+    </div>
+
+    <div>
+      <h2>Other Margarita Recipes</h2>
+      <a href="https://www.gimmesomeoven.com/cranberry-margaritas/"> Cranberry Margarita</a><br>
+      <a href="https://www.liquor.com/recipes/margarita/"> Classic Margarita</a>
+    </div>
+  </div>
+
+  <script src="script.js"></script>
+  <!--
+  This script places a badge on your repl's full-browser view back to your repl's cover
+  page. Try various colors for the theme: dark, light, red, orange, yellow, lime, green,
+  teal, blue, blurple, magenta, pink!
+  -->
+  <script src="https://replit.com/public/js/replit-badge.js" theme="blue" defer></script>
```

Comment:
Is this script used at all? If you remove it does it change the look of your page? If it is not used you can remove the comment and script tag

Reply:
Good afternoon, that's a good question the link is to Replit i will check to see if I need it